cf-usb/config-production.yaml

```diff
@@ -0,0 +1,12 @@
+src-repo: https://github.com/SUSE/cf-usb-sidecar.git
+src-branch: develop
+
+ci-repo: https://github.com/SUSE/cf-ci.git
+ci-branch: master
+
+docker-server: ""
+docker-tag: latest
+
+s3-bucket: cf-usb-release-blobstore
```

Comment:
I'm not sure this is the correct bucket to put things into.
Jan thinks we should stick it in our general catch-all bucket until we have something more organized.